src/main/resources/view/PersonDetailCard.fxml

```diff
@@ -7,39 +7,49 @@
 <?import javafx.scene.layout.GridPane?>
 <?import javafx.scene.layout.HBox?>
 <?import javafx.scene.layout.RowConstraints?>
+<?import javafx.scene.layout.StackPane?>
 <?import javafx.scene.layout.VBox?>
-
-<HBox id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
-  <GridPane HBox.hgrow="ALWAYS">
-    <columnConstraints>
-      <ColumnConstraints hgrow="SOMETIMES" minWidth="10" prefWidth="150" />
-    </columnConstraints>
-    <VBox alignment="CENTER_LEFT" minHeight="105" spacing="12" GridPane.columnIndex="0">
-      <padding>
-        <Insets bottom="5.0" left="15.0" right="15.0" top="5.0" />
-      </padding>
-      <Label fx:id="name" styleClass="label-header" text="\$first" />
-      <FlowPane fx:id="tags" />
-        <HBox spacing="24.0" VBox.vgrow="ALWAYS">
-            <VBox>
-                <Label styleClass="label-title" text="Phone" />
-                <Label fx:id="phone" styleClass="label-content" text="\$phone" />
+<HBox xmlns:fx="http://javafx.com/fxml/1" id="cardPane" fx:id="cardPane"
+      xmlns="http://javafx.com/javafx/17">
+    <GridPane HBox.hgrow="ALWAYS">
+        <columnConstraints>
+            <ColumnConstraints hgrow="SOMETIMES" minWidth="10" prefWidth="150"/>
+        </columnConstraints>
+        <VBox>
+            <VBox alignment="CENTER_LEFT" minHeight="105" spacing="12" GridPane.columnIndex="0">
```

Comment:
I think the min height here might not be enough? The address part of it is cut off

<img width="997" alt="Screenshot 2024-10-24 at 10 01 30 AM" src="https://github.com/user-attachments/assets/ac6aabaa-90b7-4517-af9c-6fd918c6f063">

Reply:
just a thought, perhaps it will be more user friendly if the entire right pane scrolls when overflowed instead of just the list of logs, but that can be done in increment v1.5.

Reply:
might also be helpful to reduce the vertical padding on the list items because based on the screenshot it seems that with the minimum window size, the user can oly see one complete log record at any one point in time